### ISSUE TITLE
Conda env dependencies not found

### ISSUE THREAD

**Issue by t-neumann** [MolecularAI/Chemformer#32]
Dear Chemformer Team,

I cloned your repo and tried to set up the conda env with your env.yml:

`conda env create -f env.yaml`

However, that simply prompts a massive list of unfound packages and aborts. Is there an updated / more lenient version of the `env.yaml` or some container you could offer?

Thanks a lot - Tobi

```
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - libgomp==12.2.0=h65d4601_19
  - pillow==9.2.0=py37h850a105_2
  - pixman==0.40.0=h36c2ea0_0
  - fontconfig==2.14.2=h14ed4e7_0
  - mkl==2021.4.0=h06a4308_640
  - libstdcxx-ng==12.2.0=h46fd767_19
  - libuuid==2.32.1=h7f98852_1000
  - pcre==8.45=h9c3ff4c_0
  - intel-openmp==2021.4.0=h06a4308_3561
  - xorg-libice==1.0.10=h7f98852_0
  - numpy==1.21.5=py37h6c91a56_3
  - libopenblas==0.3.21=pthreads_h78a6416_3
  - _libgcc_mutex==0.1=conda_forge
  - sqlite==3.40.0=h4ff8645_0
  - python==3.7.11=h12debd9_0
  - xorg-libxdmcp==1.1.3=h7f98852_0
  - readline==8.2=h8228510_1
  - libglib==2.68.4=h3e27bee_0
  - libpng==1.6.39=h753d276_0
  - libwebp-base==1.3.0=h0b41bf4_0
  - lcms2==2.14=h6ed2654_0
  - brotlipy==0.7.0=py37h27cfd23_1003
  - boost-cpp==1.74.0=h312852a_4
  - icu==68.2=h9c3ff4c_0
  - certifi==2022.12.7=py37h06a4308_0
  - libdeflate==1.14=h166bdaf_0
  - xorg-renderproto==0.11.1=h7f98852_1002
  - openjpeg==2.5.0=h7d73246_1
  - libuv==1.44.2=h5eee18b_0
  - xorg-kbproto==1.0.7=h7f98852_1002
  - cairo==1.16.0=h6cf1ce9_1008
  - expat==2.5.0=h27087fc_0
  - pyopenssl==23.0.0=py37h06a4308_0
  - xorg-xextproto==7.3.0=h0b41bf4_1003
  - pysocks==1.7.1=py37_1
  - freetype==2.12.1=hca18f0e_1
  - urllib3==1.26.14=py37h06a4308_0
  - lerc==4.0.0=h27087fc_0
  - libgfortran5==12.2.0=h337968e_19
  - libffi==3.3=h58526e2_2
  - libsqlite==3.40.0=h753d276_0
  - tk==8.6.12=h27826a3_0
  - cffi==1.15.1=py37h74dc2b5_0
  - xorg-libxrender==0.9.10=h7f98852_1003
  - numpy-base==1.21.5=py37ha15fc14_3
  - bzip2==1.0.8=h7f98852_4
  - libiconv==1.17=h166bdaf_0
  - libzlib==1.2.13=h166bdaf_4
  - ncurses==6.3=h27087fc_1
  - ca-certificates==2023.05.30=h06a4308_0
  - mkl-service==2.4.0=py37h7f8727e_0
  - cudatoolkit==11.1.74=h6bb024c_0
  - xorg-libx11==1.8.4=h0b41bf4_0
  - libxcb==1.13=h7f98852_1004
  - xorg-libxext==1.3.4=h0b41bf4_2
  - boost==1.74.0=py37h796e4cb_5
  - libgcc-ng==12.2.0=h65d4601_19
  - zlib==1.2.13=h166bdaf_4
  - mkl_fft==1.3.1=py37hd3c417c_0
  - xorg-libxau==1.0.9=h7f98852_0
  - setuptools==59.8.0=py37h89c1867_1
  - cryptography==39.0.1=py37h9ce1e76_0
  - _openmp_mutex==4.5=2_gnu
  - openssl==1.1.1u=h7f8727e_0
  - xz==5.2.6=h166bdaf_0
  - ninja==1.10.2=h06a4308_5
  - pandas==1.3.5=py37he8f5f7f_0
  - rdkit==2020.09.1=py37h0c252aa_0
  - zstd==1.5.2=h3eb15da_6
  - jpeg==9e=h0b41bf4_3
  - ld_impl_linux-64==2.40=h41732ed_0
  - gettext==0.21.1=h27087fc_0
  - mkl_random==1.2.2=py37h51133e4_0
  - libgfortran-ng==12.2.0=h69a702a_19
  - idna==3.4=py37h06a4308_0
  - pycairo==1.21.0=py37h0afab05_1
  - pthread-stubs==0.4=h36c2ea0_1001
  - requests==2.28.1=py37h06a4308_0
  - xorg-libsm==1.2.3=hd9c2040_1000
  - libgcc==7.2.0=h69d50b8_2
  - ninja-base==1.10.2=hd09550d_5
  - xorg-xproto==7.0.31=h7f98852_1007
  - libtiff==4.4.0=h82bc61c_5
```

**Comment by t-neumann** [comment]
Maybe I should mention I'm trying to get it installed on a MacBook

**Comment by t-neumann** [comment]
OK found the reason listed in closed issue #31
